tests/mlmodel_langchain/test_vectorstore.py

```diff
@@ -0,0 +1,165 @@
+            "vendor": "openAI",
+            "ingest_source": "Python",
+        },
+    ),
+    (
+        {"type": "LlmEmbedding", "timestamp": 1702052394882},
+        {
+            "id": None,  # UUID that changes with each run
+            "appName": "Python Agent Test (mlmodel_langchain)",
+            "span_id": None,
+            "trace_id": "trace-id",
+            "request_id": "d5d71019880e25a94de58b927045a202",
+            "transaction_id": "transaction-id",
+            "input": "[[12833, 420, 11914, 25, 22691]]",
+            "api_key_last_four_digits": "sk-CRET",
+            "duration": None,  # Changes with every run
+            "request.model": "text-embedding-ada-002",
+            "response.model": "text-embedding-ada-002-v2",
+            "response.organization": "new-relic-nkmd8b",
+            "response.api_type": "None",
+            "response.usage.total_tokens": 5,
+            "response.usage.prompt_tokens": 5,
+            "response.headers.llmVersion": "2020-10-01",
+            "response.headers.ratelimitLimitRequests": 3000,
+            "response.headers.ratelimitLimitTokens": 1000000,
+            "response.headers.ratelimitResetTokens": "0s",
+            "response.headers.ratelimitResetRequests": "20ms",
+            "response.headers.ratelimitRemainingTokens": 999994,
+            "response.headers.ratelimitRemainingRequests": 2999,
+            "vendor": "openAI",
+            "ingest_source": "Python",
+        },
+    ),
+    (
+        {"type": "LlmVectorSearch", "timestamp": 1702052394890},
+        {
+            "span_id": "span-id",
+            "trace_id": "trace-id",
+            "transaction_id": "transaction-id",
+            "id": None,  # UUID that changes with each run
+            "vendor": "LangChain",
+            "request.query": "Complete this sentence: Hello",
+            "request.k": 1,
+            "duration": None,  # Changes with every run
+            "response.number_of_documents": 1,
+        },
+    ),
+    (
+        {"type": "LlmVectorSearchResult", "timestamp": 1702052424031},
+        {
+            "search_id": None,  # UUID that changes with each run
+            "sequence": 0,
+            "page_content": "Hello world!\n1",
+            "span_id": "span-id",
+            "trace_id": "trace-id",
+            "transaction_id": "transaction-id",
+            "id": None,  # UUID that changes with each run
+            "vendor": "LangChain",
+            "request.query": "Complete this sentence: Hello",
+            "request.k": 1,
+            "duration": None,  # Changes with every run
+            "response.number_of_documents": 1,
+            "metadata.source": "/__w/newrelic-python-agent/newrelic-python-agent/tests/mlmodel_langchain/hello.pdf",
+            "metadata.page": 0,
+        },
+    ),
+]
+
+
+@reset_core_stats_engine()
+@validate_custom_events(vectorstore_recorded_events)
+@validate_custom_event_count(count=4)
+# @validate_transaction_metrics(
```

Comment:
We should be creating a metric for this in our vector store monitoring and testing for it here. You just need to add in the following call into the wrapper instrumentation function:
```
transaction.add_ml_model_info("Langchain", LANGCHAIN_VERSION)

```